modules/os2forms_digital_post/src/Helper/ForsendelseHelper.php

```diff
@@ -24,6 +27,22 @@ class ForsendelseHelper extends AbstractMessageHelper {
   // PostKategoriKode.
   public const POST_KATEGORI_KODE_PRIORITAIRE = 'Prioritaire';
 
+  /**
+   * {@inheritDoc}
+   */
+  // phpcs:disable Generic.CodeAnalysis.UselessOverridingMethod
```

Comment:
Why don't you add the attributes in `AbstractMessageHelper`?

Reply:
Doh! Updated..